jenkins/pipeline.groovy

```diff
@@ -121,11 +114,13 @@ pipeline {
                 }
 
                 sh 'echo Prepare: \$(date -u "+%s")'
-                git branch: '5.7', url: 'https://github.com/Percona-Lab/ps-build'
+                git branch: 'fix-8.0-version', url: 'https://github.com/Percona-Lab/ps-build'
                 sh '''
-                    git reset --hard
-                    git clean -xdf
+                    sudo git reset --hard
```

Comment:
Maybe a good idea to put the explanation for sudo here in a comment

Reply:
fixed

Reply:
fixed